### PR TITLE
Add KooZic

### PR DESCRIPTION
**Pull request by DocMarty84**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -955,6 +955,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [gonic](https://github.com/sentriz/gonic) - Lightweight music streaming server. Subsonic compatible. `GPL-3.0` `Go`
 - [Groove Basin](https://github.com/andrewrk/groovebasin) - Music player server with a web-based user interface inspired by Amarok 1.4. `MIT` `Nodejs`
 - [koel](http://koel.phanan.net/) - Personal music streaming server that works. ([Source Code](https://github.com/phanan/koel)) `MIT` `PHP`
+- [KooZic](https://koozic.net/) - Music server with powerful playlist features. Subsonic compatible ([Demo](https://demo.koozic.net/public), [Source Code](https://github.com/DocMarty84/koozic)) `LGPL` `MIT` `Python`
```
Comment (n8225) [inline README.md]: I didn't see a reference to the MIT license in https://github.com/DocMarty84/koozic/blob/v3/LICENSE in which case change
```
 `LGPL` `MIT` `Python` 
```
to
```
`LGPL-3.0` `Python`
```
If MIT does need to be included it should be 
```
`LGPL-3.0/MIT` `Python`
```

**Comment by DocMarty84** [comment]
Thanks for the prompt reply. The MIT license applies to 2 submodules (oomusic and oovideo): https://github.com/DocMarty84/koozic/tree/v3/addons

The license is here:
https://github.com/DocMarty84/oomusic
https://github.com/DocMarty84/oovideo

I updated the license display as you requested.

File: README.md [hunk 2]
```diff
@@ -955,6 +955,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [gonic](https://github.com/sentriz/gonic) - Lightweight music streaming server. Subsonic compatible. `GPL-3.0` `Go`
 - [Groove Basin](https://github.com/andrewrk/groovebasin) - Music player server with a web-based user interface inspired by Amarok 1.4. `MIT` `Nodejs`
 - [koel](http://koel.phanan.net/) - Personal music streaming server that works. ([Source Code](https://github.com/phanan/koel)) `MIT` `PHP`
+- [KooZic](https://koozic.net/) - Music server with powerful playlist features. Subsonic compatible ([Demo](https://demo.koozic.net/public), [Source Code](https://github.com/DocMarty84/koozic)) `LGPL-3.0/MIT` `Python`
```
Comment (n8225) [inline README.md]: My apologies for missing this earlier, but your missing a `.` at the end of the description. 
Could you update the description to read something along the lines of "Music server with powerful playlist features and subsonic compatibility.

Comment (DocMarty84) [reply README.md]: Done :smile:

**Pull request merged**